src/utils/__tests__/flattenTypes.spec.ts

```diff
@@ -5,13 +5,6 @@ describe('flattenTypes util', () => {
     expect(flattenTypes(type)).toEqual(type);
   });
 
-  it('returns undefined when no valid type is found', () => {
-    expect(flattenTypes(2)).toBeUndefined();
-    expect(flattenTypes(void 0)).toBeUndefined();
-    expect(flattenTypes('foo')).toBeUndefined();
-    expect(flattenTypes(['test', 'foo'])).toBeUndefined();
-  });
-
```

Comment:
why deleted?

Reply:
Dupe, see the test above it.

Reply:
Likely to be the result of incorrect merging.

Reply:
~but it's not a dupe.~ ah. the test below is different but better. so yes a dupe.